### ISSUE TITLE
login in behat fails sometimes on travis-ci

### ISSUE THREAD

**Issue by Ma27** [Sententiaregum/Sententiaregum#205]
following error appears:

```
Given I'm logged in as "Ma27" with password "72aM" # src/AppBundle/Features/language/locale.feature:19
```

**Comment by Ma27** [comment]
caused by an error inside the ip tracker. The old behat code was quite buggy, refactored a lot of that in #35 in order to get a green and working test suite.

**Comment by Ma27** [comment]
ok testsuite green so far, just hhvm fails due to this: https://github.com/facebook/hhvm/issues/4277

**Comment by Ma27** [comment]
will be fixed in #35, closing now.
